DANS-KNAW/easy-convert-bag-to-deposit
# DD-857 + DD-835: keep date published

Fixes DD-857 + DD-835: keep date published

When applied it will
--------------------
* no longer clear the first published date in the AMD
* 
* 

Where should the reviewer @DANS-KNAW/easy start?
------------------------------------------------

How should this be manually tested?
-----------------------------------
* [x] ran `mvn clean install` witout `-DskipTests `
* [x] download the bag attached to the issue and execute

      run.sh -u data/DD-857/602dd058-a1a9-4c48-b877-23e4960a403e -t DOI -s FEDORA
   
  Compare the changes in the `amd.xml`. That should be at most be different quotes and order for attributes.

Related pull requests on github
-------------------------------

repo                       | PR                | note
-------------------------- | ----------------- | ----
easy-                      | [PR#](PRlink)     | ..
